sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -227,6 +227,13 @@ private[spark] object SQLConf {
     defaultValue = Some(true),
     doc = "<TODO>")
 
+  val PARQUET_MERGE_SCHEMA_ENABLED = booleanConf("spark.sql.parquet.mergeSchema",
+    defaultValue = Some(true),
+    doc = "Turn on the schema merge feature of parquet datasource API. " +
+          "Enable it will merge different schema of parquet" +
+          "Disable it will spped up the parquet schema loading time if all your parquet " +
+          "schema is the same" )
```

Comment:
Please reword `doc` as following:

> When true, the Parquet data source merges schemas collected from all data files, otherwise the schema is picked from the summary file or a random data file if no summary file is available.